packages/jest-circus/src/index.ts

```diff
@@ -63,7 +63,21 @@ const _dispatchDescribe = (
     mode,
     name: 'start_describe_definition',
   });
-  blockFn();
+  const describeReturn = blockFn();
+
+  // TODO throw in Jest 25
+  if (isPromise(describeReturn)) {
+    console.warn(
```

Comment:
Please include a stack trace or something so it's easier for people to track down which `describe` this is.